kong/consumer_group_service.go

```diff
@@ -63,7 +63,7 @@ func (s *ConsumerGroupService) Get(ctx context.Context,
 		return nil, fmt.Errorf("nameOrID cannot be nil for Get operation")
 	}
 
-	endpoint := fmt.Sprintf("/consumer_groups/%v", *nameOrID)
+	endpoint := fmt.Sprintf("/consumer_groups/%v?list_consumers=false", *nameOrID)
```

Comment:
So according to https://github.com/Kong/go-kong/pull/488/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR82-R83 this is only available in GW 3.9+. What will happen when we provide these query params for older versions?

Reply:
The query param is ignored in gw <3.9
I have added a test case for the same too to ensure that things don't break with older versions.